service-base/src/main/java/org/eclipse/hono/service/AbstractServiceBase.java

```diff
@@ -33,6 +35,14 @@
 
     private HttpServer healthCheckServer;
 
+    private ApplicationConfigProperties applicationConfig;
```

Comment:
I think we should initalize this with a `new ApplicationConfigProperties()` to prevent *no null* checks ...